default.nix

```diff
@@ -16,6 +16,9 @@
   mixins-systemd-boot = import ./mixins/systemd-boot.nix;
   mixins-telegraf = import ./mixins/telegraf.nix;
 
+  # Modules
+  github-runner = import ./modules/github-runners.nix;
```

Comment:
If the module is getting upstreamed, let's keep it private and import it in the role directly.